.changeset/light-taxis-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/typography': patch
```

Comment:
In the ticket you mentioned that this will be a major change, why the switch to patch?

Reply:
Ah, missed that. Will fix 👍

Reply:
It should always have been `inline`, so this is just a bug fix. Not sure why I wrote that in the ticket